src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
+    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
```

Comment:
@joperezr this is the ref which only has one build the latest netstandard1.7 version, so I think the ProjectReference @stephentoub has will work.